### PR TITLE
Windows environment

### PR DESCRIPTION
**Pull request by Stoltefar**
New attempt with Jenkinsfile restored .

### CODE REVIEW

**Review by LiibanAbdulkadir — APPROVED** [review]
Perfekt!

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@
 log.html
 *output.xml
 *report.html
-*selenium-screenshot*.png
\ No newline at end of file
+*selenium-screenshot*.png
```
Comment (TobiasBjorlin) [inline .gitignore:6]: Bra

**Review by TobiasBjorlin — APPROVED** [review]
Nice work!

File: .gitignore (hunk 1 above)
Comment (juheejulia) [inline .gitignore:6]: Bra!

**Review by juheejulia — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (LisaOla) [inline .gitignore:6]: Bra!

**Review by LisaOla — APPROVED** [review]

**Pull request merged**